plugins/meta/flannel/flannel_linux_test.go

```diff
@@ -50,8 +54,8 @@ var _ = Describe("Flannel", func() {
 {
   "name": "cni-flannel",
   "type": "flannel",
-	"subnetFile": "%s",
-	"dataDir": "%s"%s
+    "subnetFile": "%s",
```

Comment:
This is unnecessary and incorrect change in formatting. Please use gofmt.

Reply:
Done